install_custom_module_map/after/Example.xcodeproj

```diff
@@ -53,7 +53,7 @@ Targets:
         Shell Path: "/bin/sh"
         Shell Script: '"${PODS_ROOT}/Target Support Files/Pods-Example/Pods-Example-frameworks.sh"
 
-'
+          '
```

Comment:
drcapulet do you think this is related to your recent change for pretty printing script phases?